library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -186,6 +195,8 @@ private void init(@Nullable AttributeSet attributeSet) {
             @Override
             public void onClick(View v) {
                 focusOnUserLocation(true);
+                hasAlreadyRequestedEnableLocation = false;
```

Comment:
Why is this necessary?

Reply:
KujakuMapView **has already requested to enable location**